app/assets/stylesheets/header.scss

```diff
@@ -43,6 +43,11 @@ body > header {
       background: none;
       color: $highlight-white;
     }
+    &:focus {
+        color: $black;
+        outline: thin dotted;
+        text-decoration: none;
+    }
```

Comment:
Hm, because it's WIP...(No no, I don't try to find excuses ;) ). I'll do that soon.